data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/NetworkElementParent.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_api;
+
+/**
+ * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
+ */
+public interface NetworkElementParent {
+    /**
+     * Add a (or set the) network element using a {@code NetworkElement} object
+     * @param networkElement: {@code NetworkElement}
+     * @return the added {@code NetworkElement} object
+     */
+    NetworkElement addNetworkElement(NetworkElement networkElement);
```

Comment:
I would rather put newNetworkElement method here, because it is common to all NetworkElementParent implementations